flash_patcher/compile/compilation.py

```diff
@@ -1,12 +1,14 @@
 from __future__ import annotations
 
 import base64
-from logging import error, info
+from logging import getLogger
 from pathlib import Path
 
 from flash_patcher.compile.ffdec import FFDecInterface
 from flash_patcher.exception.dependency import DependencyError
 
+logger = getLogger(__name__)
```

Comment:
If possible, we should avoid needing to add this line everywhere we ask for the logger. I'll look into whether we can use a global variable for our logger instead

Reply:
We can move the logger to its own file as follows:

`util/logging.py`:

```
from logging import getLogger

logger = getLogger(__name__)
```

We can just run `from flash_patcher.util.logging import logger` to have a logger.

This has the advantage of allowing us to better manage the logger configuration, and not creating global variables.

Reply:
I'm not aware of any way to make module-wide global variables. If you find a way, I'd be happy to implement it.

Reply:
See my comment above.

We can create the logger as a variable in a file called `util/logging.py` and directly import the logger as a variable where it's needed.

If we need to initialize it, we can do that in `patcher.py -> main()` and have a function in `logging.py` to initialize it.

Reply:
> See my comment above.

Oops. I think I replied before that comment had loaded. Or maybe I just missed it.

That solution works for me. I'll look into implementing it.